timescaledb/tutorials/aws-lambda/continuous-deployment.md

```diff
@@ -0,0 +1,180 @@
+* Git ([installation options here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git))
+* GitHub CLI tool ([installation options here](https://github.com/cli/cli#installation))
+* AWS account
+
+## Procedure: Creating a new Lambda function in AWS console
+1.  Create a new Lambda function called `lambda-cd` by navigating to the AWS Lambda console and creating a new function:
+    ![create new function](https://assets.timescale.com/docs/images/tutorials/aws-lambda-tutorial/create_new_function.png)
+1.  Add `lambda-cd` as your function name, choose your preferred runtime environment, and click `Create function`:
+    ![create lambda from scratch](https://assets.timescale.com/docs/images/tutorials/aws-lambda-tutorial/from_scratch.png)
+    Take a note of the function name, you need it later to configure the deployment file.
+
+## Procedure: Creating a new GitHub repository
+
+Now let's create a new GitHub repository which contains the function code.
+
+1.  Create a new GitHub [repository](https://github.com/new).
+    ![lambda github repo](image url)
+1.  On your computer, create a new project folder called `lambda-cd`:
+    ```bash
+    mkdir lambda-cd
+    cd lambda-cd
+    ```
+1.  Create the Lambda function that you want to upload.
+    As an example, here's a Python Lambda function which returns data from a TimescaleDB table called `stocks_intraday`
+    ([read the tutorial to build a TimescaleDB API with Lambda here](/tutorials/aws-lambda/create-data-api/)).
+    ```bash
+    touch function.py
+    ```
+
+    ```python
+    # function.py:
+    import json
+    import psycopg2
+    import psycopg2.extras
+    import os
+    
+    def lambda_handler(event, context):
+        db_name = os.environ['DB_NAME']
+        db_user = os.environ['DB_USER']
+        db_host = os.environ['DB_HOST']
+        db_port = os.environ['DB_PORT']
+        db_pass = os.environ['DB_PASS']
+        
+        conn = psycopg2.connect(user=db_user, database=db_name, host=db_host,
+                                password=db_pass, port=db_port)
+        
+        sql = "SELECT * FROM stocks_intraday ORDER BY time DESC LIMIT 10"
+        cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+        cursor.execute(sql)
+        result = cursor.fetchall()
+        
+        return {
+            'statusCode': 200,
+            'body': json.dumps(list_of_dicts, default=str),
+            'headers': {
+                "Content-Type": "application/json"
+            }
+        }
+    ```
+1.  Initialize a git repository and push the project to GitHub:
+    ```bash
+    git init
+    git add function.py
+    git commit -m "Initial commit: add Lambda function"
+    git branch -M master
+    git remote add origin <YOUR_GITHUB_PROJECT_URL.git>
+    git push -u origin master
+    ```
+
+At this point, you have a GitHub repo with just the Lambda function in it. Let's see how to connect this repo to the AWS Lambda function.
+
+## Connect GitHub and AWS Lambda
+
```

Comment:
This needs a sentence.

Reply:
https://github.com/timescale/docs/pull/418/commits/dbee9ec713e0d08e4a7080750b46ce46e587925e